presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -724,6 +736,13 @@ void PrestoServer::stop() {
   }
 }
 
+void PrestoServer::enableAnnouncer(bool enable) {
+  if (announcer_ != nullptr) {
+    LOG(INFO) << (enable ? "Enabling announcer..." : "Disabling announcer...");
```

Comment:
If we have log in enableRequest, then we don't need log here.

Reply:
He-he, sure, I can move. :)